docsrs_bindings.rs

```diff
@@ -2011,105 +2092,122 @@ pub struct _php_stream {
 }
 impl _php_stream {
     #[inline]
-    pub fn is_persistent(&self) -> u8 {
-        unsafe { ::std::mem::transmute(self._bitfield_1.get(0usize, 1u8) as u8) }
+    pub fn is_persistent(&self) -> u16 {
```

Comment:
Are all the changes to the bindings intentional? This looks like maybe you didn't compile it on the correct PHP version / system.

Reply:
Hmmm, I used php 8.3 build from source with debug enabled on a 64bit machine.

Reply:
Ah ok, I guess that got updated in https://github.com/davidcole1340/ext-php-rs/commit/f621e60be66425d3bf3ac1a5ebccf675cdda263c